index.html

```diff
@@ -0,0 +1,276 @@
+          <section class="column">
+            <div class="cards">
+              <img src="Images/vaquero.jpg" alt="speaker4" />
+              <div class="people-info">
+                <h3 class="speaker">Arturo Ramirez</h3>
+                <h4 class="red-subtitle italic">Cowboy</h4>
+                <p class="info">
+                  Miguel is the most coveted and qualified cowboy of the state.
+                  He is the only one capable of making a group of horses dance
+                  in tune.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/poncho.jpg" alt="speaker5" />
+              <div class="people-info">
+                <h3 class="speaker">Alonso Mora</h3>
+                <h4 class="red-subtitle italic">Show Organizer</h4>
+                <p class="info">
+                  Alonso is the best show organizer of the country· Big
+                  companies have request his organizer services because they
+                  know that his shows that he prepares are incredible. You will
+                  never get bored in one of his shows.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/amram2.jpg" alt="speaker6" />
+              <div class="people-info">
+                <h3 class="speaker">Amram Martinez</h3>
+                <h4 class="red-subtitle italic">Security</h4>
+                <p class="info">
+                  Amram is the responsable of all the security system. Having a
+                  military training is his powerful weapon. His team is prepared
+                  for all kinds of mishaps.
+                </p>
+              </div>
+            </div>
+          </section>
+        </div>
+        <button type="button" class="show-more disapeer">Show more</button>
+      </section>
+      <section class="patrocinadores">
+        <h3 class="speaker-title hide">Partner</h3>
+        <div class="logos">
+          <img
+            class="hide"
+            src="Images/cocacola.png"
+            alt="cocacola-logo"
+            height="100"
+          />
+          <img
+            class="hide"
+            src="Images/indio.png"
+            alt="indio-logo"
+            height="100"
+          />
+          <img
+            class="hide"
+            src="Images/bimbo.png"
+            alt="bimbo-logo"
+            height="100"
+          />
+          <img
+            class="hide"
+            src="Images/blachere.png"
+            alt="blachere-logo"
+            height="100"
+          />
+        </div>
+      </section>
+    </main>
+    <footer class="footer">
+      <img class="hide" src="Images/logo-villa.png" alt="logo" height="40" />
+      <div class="hide">
+        <p class="rights">2021 Villa Iluminada. Rights Reserved.</p>
+        <p class="rights">This project was made by Ivan Linares.</p>
+      </div>
+    </footer>
```

Comment:
The **background color** and **font-weight**/ **font size** of your footer on both the mobile and desktop versions are the same on the _about us page_. As specified let's ensure to improve upon these aspects:
- [ ]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.
- [ ] Adjust the **font-weight**/ **font size** of the text in this section to match that of the provided template design